src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -499,14 +525,15 @@ const ComposeForm = props => {
       await setSendMessageFlag(false);
       const validSignatureNotRequired = messageValid && !isSignatureRequired;
       const validSignatureRequired =
-        isSignatureRequired && signatureValid && checkboxValid;
+        isSignatureRequired && signatureValid && checkboxValid && messageValid;
```

Comment:
I am wondering if renaming `validSignatureRequired` to `validSignature` or `isSignatureValid` ay be more comprehensive? I am just finding it a bit confusing when contrasted with `validSignatureNotRequired`.